### ISSUE TITLE
shufflenet v2

### ISSUE THREAD

**Issue by darcula1993** [keras-team/keras-applications#199]
Is there any plan to provide shufflenet v2 model and pre-trained weights?
Thank you very much for any response!
